packages/l/llvm-15/package.yml

```diff
@@ -0,0 +1,179 @@
+name       : llvm-15
+version    : 15.0.7
+release    : 1
+source     :
+    - https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.7/llvm-project-15.0.7.src.tar.xz : 8b5fcb24b4128cf04df1b0b9410ce8b1a729cb3c544e6da885d234280dedeac6
+homepage   : http://llvm.org/
+license    : Apache-2.0-with-LLVM-exception
+summary    : Low Level Virtual Machine (Reusable compiler and toolchains)
+component  : programming.devel
+debug      : no
+libsplit   : no
+clang      : yes
+description: |
+    The LLVM package contains a collection of modular and reusable compiler and toolchain technologies. The Low Level Virtual Machine (LLVM) Core libraries provide a modern source and target-independent optimizer, along with code generation support for many popular CPUs (as well as some less common ones!). These libraries are built around a well specified code representation known as the LLVM intermediate representation ("LLVM IR").
+# patterns   : /*
+patterns   :
+    - ^llvm-clang-15-devel :
+        - /usr/lib64/llvm15/include/clang*
+        # - /usr/lib64/llvm15/include/lld*
+        - /usr/lib64/llvm15/lib64/libclang*.a
+        # - /usr/lib64/llvm15/lib64/liblld*.a
+        - /usr/lib64/llvm15/lib64/cmake/clang
+        # - /usr/lib64/llvm15/lib64/cmake/lld
+        - /usr/lib64/llvm15/share/clang
+    - ^llvm-clang-15 :
+        - /usr/bin/clang*
+        - /usr/lib64/llvm15/bin/clang*
+        # - /usr/bin/clang*
```

Comment:
This commented patterns look like they are leftovers from testing, can they be removed?